src/utils/asyncStore/identityStorage.js

```diff
@@ -89,30 +89,39 @@ export const updateClaimCategories = async (updatedCategories) => {
   }
 };
 
-export const storeSeedClaims = async (identityId) => {
-  const seededClaims = claims(identityId);
-
+export const getClaims = async () => {
   try {
-    await AsyncStorage.setItem('claims', JSON.stringify({ claims: seededClaims }));
+    const result = await AsyncStorage.getItem('claims');
+    return result ? JSON.parse(result).claims : [];
   } catch (error) {
     console.log(error);
```

Comment:
You either need to return some error or throw an error. You cannot just swallow the exception.